src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -592,6 +592,42 @@ void ClusterResourceScheduler::DeleteResource(const std::string &node_id_string,
   }
 }
 
+std::string ClusterResourceScheduler::SerializedTaskResourceInstances(
```

Comment:
This won't be maintainable, can we use boost::json or other json library?

Reply:
Here we need to distinguish which resource is instance.

For example, a task request two cpu core. If `CPU` is instance resource , the allocation resource will be `{"CPU":[10000,0,10000]}`. If `CPU` is not instance resource, the allocation resource will be `{"CPU":20000}`